### PR TITLE
Fix .travis.yml

### PR DESCRIPTION
**Pull request by andreasnoack**
Also fix the grouping tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,21 +4,25 @@ os:
...
-  - julia -e "using Pkg; Pkg.add(PackageSpec(name = "Pumas", url = "http://github.com/PumasAI/Pumas.jl", rev = "master"))"
+  - julia -e 'using Pkg; Pkg.add(PackageSpec(url="http://github.com/PumasAI/Pumas.jl", rev="master"))'
+script:
+  julia --color=yes -e 'using Pkg; Pkg.build(verbose=true)'
+  julia --color=yes -e "using Pumas'
+  julia --color=yes --check-bounds=yes -e 'using Pkg; Pkg.test(coverage=true)'
 notifications:
   email: false
 jobs:
   include:
```
Comment (asinghvi17) [inline .travis.yml]: Quotes are mismatched here

Comment (andreasnoack) [reply .travis.yml]: Just fixed it. Let's see if that is enough to avoid the timeout on Mac.

**Comment by asinghvi17** [comment]
Huh, looks like a Pumas issue is causing failures on the nightly. 

```
$ julia --color=yes -e 'using Pumas'
WARNING: could not import Compiler.just_construct_ssa into Wrap
```

**Comment by asinghvi17** [comment]
Thanks a lot, @andreasnoack!

**Comment by andreasnoack** [comment]
It's still on the edge of timing out. I have an idea. Will open a followup PR.

**Pull request merged**